src/main/java/ee/carlrobert/codegpt/util/DiffUtils.java

```diff
@@ -0,0 +1,52 @@
+package ee.carlrobert.codegpt.util;
+
+import com.intellij.diff.chains.SimpleDiffRequestChain;
+import com.intellij.diff.editor.ChainDiffVirtualFile;
+import com.intellij.diff.impl.CacheDiffRequestChainProcessor;
+import com.intellij.diff.requests.SimpleDiffRequest;
+import com.intellij.openapi.fileEditor.ex.FileEditorManagerEx;
+import com.intellij.openapi.fileEditor.impl.EditorComposite;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.Key;
+import org.jetbrains.annotations.NotNull;
+
+
+public class DiffUtils {
+
+
+    public static final String DIFF_TITLE = "CodeGPT Diff";
```

Comment:
Titles and labels are actually stored in `src/main/resources/messages/codegpt.properties`